addons/udes_priorities/models/priority.py

```diff
@@ -26,17 +26,36 @@ class UdesPriority(models.Model):
         default=lambda self: uuid4(),
     )
 
+    @api.multi
+    @api.depends("reference")
+    def _compute_outstanding_pickings(self):
+        Picking = self.env["stock.picking"]
+        for record in self:
+            record.outstanding_pickings = Picking.search_count(
+                [
+                    ("priority", "=", record.reference),
+                    ("state", "not in", ("cancel", "done")),
+                ]
+            )
+
     def _default_sequence(self):
         max_rec = self.search([], order="sequence desc, id desc", limit=1)
         return max_rec.sequence + 1 if max_rec else 1
 
-    sequence = fields.Integer(default=_default_sequence, help="Used to order the priorities")
+    sequence = fields.Integer(
+        default=_default_sequence, help="Used to order the priorities"
+    )
     description = fields.Text(help="A description about the purpose of the priority")
     picking_type_ids = fields.Many2many(
         "stock.picking.type",
         string="Stock Transfer Types",
         help="The stock picking types this priority is visible on. No picking types means all picking types",
     )
+    outstanding_pickings = fields.Integer(
+        string="Affected Stock Transfers",
+        compute="_compute_outstanding_pickings",
+        help="Active stock pickings that would be affected by changes in the current priority.",
```

Comment:
Super nit: Can we change this to `Active stock transfers?`. Otherwise it's a bit inconsistent as the string says transfers but the help says pickings.

Reply:
I was considering doing that but then I noticed on `picking_type_ids` it mentions both pickings and transfer terms so I copied the style over. Probably need to fix the one above too to be consistent.